docs/source/azure/advanced/azurefunctions.rst

```diff
@@ -27,9 +27,13 @@ When deploying an Azure function the following ARM resources are required and cr
 - Application Service Plan (shared across functions)
 - Application Service (per function)
 
-A single Application Service Plan (Basic, Standard or Premium) can service a large number
-of Application Service Instances.  If you provide the same servicePlanName with all policies or
-use the default name then only new Applications will be created during deployment, all using the same
+Functions can be ran in either a dedicated Application Service Plan (Basic, Standard or Premium) or in a Consumption plan.
```

Comment:
Function can be ~ran~ ***deployed*** in either

Reply:
Mostly to avoid grammatical debate of `ran` vs `run` in this case, but I think it should be `can be run` ;)